src/tests/ftest/util/general_utils.py

```diff
@@ -244,190 +242,6 @@ def get_random_string(length, exclude=None):
     return random_string
 
 
-@fail_on(DaosApiError)
-def get_pool(context, mode, size, name, svcn=1, log=None, connect=True):
```

Comment:
Any specific reason these are removed?

Reply:
This also was not use any were and has been replaced with test_utils.pool